### PR TITLE
Features-Works|About|Contacts

### PR DESCRIPTION
**Pull request by RileyManda**
I have completed work on the Portfolio’s first section’s Works,About and Contact as per requirements stipulated: 

Here are the main changes I made: 

* Added **Recent works** section with title,cards and tags.
* Added **About ** section with title,cards and tags and background images right and left of the view.
* Added a **Contacts** section with a form,email, and image to the bottom right with social icons below email.
* Made sure the components look great and function well on different devices.
* Adjusted header padding to the correct alignment.
Please review the future features added to this project and let me know if there are any issues or suggestions for improvement.

I look forward to your feedback. 
Thank you! 
Best regards,

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -156,20 +421,176 @@ html {
   width: 30px;
 }
 
-/* Media Queries */
+.card-container {
```
Comment (Alfredbis29) [inline style.css:436]: - [ ] I noticed that the main container of the project does not have the required "display:grid" property as specified in the project requirements.
In order to maintain consistency and ensure that the project meets the necessary standards, it's important that this requirement is followed. Please make the necessary changes to the code and resubmit your pull request.

image

File: index.html [hunk 2]
```diff
@@ -24,20 +24,199 @@
...
+          
+            <div class="card-container">
+        <div class="about-card">
+            <div class="icon-container">
+                <img src="./assets/icons/languages.svg" alt="Image Placeholder">
+            </div>
+        
+            <div class="about-card-content">
+                <h3 class="about-card-title">Languages</h3>
+                <div class="about-tags">
```
Comment (Alfredbis29) [inline index.html]: - [ ] I noticed that the project requirements call for the use of "ul" and "li" elements to structure categories in "Work" section and categories in "About myself" section, using "ul" and "li" elements is a good practice when creating lists in HTML. It provides a clear and organized structure to the content, making it easier for users to read and navigate.
Kindly ensure that you use "ul" and "li" elements in those sections meet the project requirements.

image

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
#  STATUS: REQUIRED CHANGES 
 ## Hi @RileyManda  
 Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

  
 ## Highlights 
 :heavy_check_mark: Linters are green.
:heavy_check_mark: You have a good code structure.
:heavy_check_mark: Descriptive PR
 
 ## Required Changes 
- [ ] Check under the review

 ## Optional suggestions 
N/A

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
 
 Cheers and Happy coding! :clap: :clap: :clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by RileyManda** [comment]
> # STATUS: REQUIRED CHANGES
> ## Hi @RileyManda
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
> 
> 
> 
> ## Highlights
> ✔️ Linters are green. ✔️ You have a good code structure. ✔️ Descriptive PR
> 
> ## Required Changes
> * [ ]  Check under the review
> 
> ## Optional suggestions
> N/A
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
> 
> Cheers and Happy coding! 👏 👏 👏 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

I have completed iterations on the Portfolio as requested by the reviewer to meet the requirements stipulated: 
Here are the main iterations I made: 

* Replaced span in card content and utilized ul and li as per project requirements for the categories.
* Used CSS Grid in the main container that contains all of the cards and used Flexbox to lay out an individual card.

Please review the changes and let me know if there are any issues or suggestions for improvement. I look forward to your feedback. 
Thank you! 
Best regards,

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @RileyManda 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Pull request merged**